package.json

```diff
@@ -17,39 +17,47 @@
     "react",
     "react-native",
     "jsx",
-    "js"
+    "js",
+    "javascript",
+    "ts",
+    "tsx",
+    "typescript"
   ],
   "author": "typedigital",
   "license": "Unlicense",
   "scripts": {
-    "lint": "npx eslint . --config=base.js",
-    "lint-fix": "npx eslint . --config=base.js --fix",
-    "test": "eslint ./tests/*.js --config=base.js",
+    "lint": "eslint ./*.js --config=base.js",
+    "lint-fix": "eslint ./*.js --config=base.js --fix",
+    "test": "npm-run-all test:*",
     "test:base": "eslint ./tests/base/*.js --max-warnings 0 --config=base.js",
     "test:typescript": "eslint tests/typescript/*.ts --max-warnings 0 --config=tests/typescript/.eslintrc",
-    "release": "standard-version"
+    "release": "standard-version",
+    "find-unused": "eslint-find-rules -u -n -d ./base.js",
```

Comment:
is it just on my end, or does "find-unused" already find tons of unused rulesets?

Reply:
It does find a ton of unused rules, yes. So we should tackle them in the next version I guess. What do you mean? @JoeTypeDigital